python_modules/libraries/dagster-looker/dagster_looker/api/dagster_looker_api_translator.py

```diff
@@ -15,6 +20,103 @@ class LookerInstanceData:
     explores_by_id: Dict[str, LookmlModelExplore]
     dashboards_by_id: Dict[str, Dashboard]
 
+    @staticmethod
+    def build(*, sdk: Looker40SDK) -> "LookerInstanceData":
```

Comment:
In terms of responsibility, I think `fetch_looker_data()` should stays in the resource instead of implementing `build()` here.